less/viewdetails.less

```diff
@@ -178,6 +178,31 @@
     }
   }
 
+  .md-alert {
```

Comment:
rather than adding the CSS to this already big file, I would rather build a new `markdown.less` file, imported in `c2corg_ui.less`

I know the CSS is a mess, but we should start organizing it now.

Reply:
Agree, but I never write a line of CSS. Cold we do a separate issue for this?

Reply:
It's only postponing the current situation. Let's do this later.